x509util/x509util.go

```diff
@@ -861,6 +861,34 @@ func ExtractSCT(sctData *x509.SerializedSCT) (*ct.SignedCertificateTimestamp, er
 	return &sct, nil
 }
 
+// ParseSCTListIntoSCTs serializes SCTs into SCT list.
+func ParseSCTListIntoSCTs(scts []*ct.SignedCertificateTimestamp) (*x509.SignedCertificateTimestampList, error) {
+	var sctList x509.SignedCertificateTimestampList
+	sctList.SCTList = []x509.SerializedSCT{}
+	for i, sct := range scts {
+		sctData, err := EncodeSCT(sct)
+		if err != nil {
+			return nil, fmt.Errorf("error serializing SCT number %d: %s", i, err)
```

Comment:
This is largely duplicating the error message you get from EncodeSCT. See comment about folding in EncodeSCT below, which would also fix this.

Reply:
Shortened as suggested.